very_good_core/__brick__/{{project_name.snakeCase()}}/android/app/build.gradle.kts

```diff
@@ -0,0 +1,120 @@
+import java.util.Properties
+import java.io.FileInputStream
+
+plugins {
+    id("com.android.application")
+    id("kotlin-android")
+    id("dev.flutter.flutter-gradle-plugin")
+}
+
+val localProperties = Properties()
+val localPropertiesFile = rootProject.file("local.properties")
+if (localPropertiesFile.exists()) {
+    localPropertiesFile.bufferedReader(Charsets.UTF_8).use { reader ->
+        localProperties.load(reader)
+    }
+}
+
+var flutterVersionCode = localProperties.getProperty("flutter.versionCode")
+if (flutterVersionCode == null) {
+    flutterVersionCode = "1"
+}
+
+var flutterVersionName = localProperties.getProperty("flutter.versionName")
+if (flutterVersionName == null) {
+    flutterVersionName = "1.0"
+}
```

Comment:
Do we indeed still need these? Or should we reference them? The old gradle file did have them used.

Reply:
No need for this any longer. The `defaultConfig` uses `flutter.versionCode` and `flutter.versionName`.